packages/ra-ui-materialui/src/layout/Notification.js

```diff
@@ -25,9 +25,16 @@ class Notification extends React.Component {
     state = {
         open: false,
     };
+    componentWillMount = () => {
+        this.setNextState();
```

Comment:
shouldn't you use `this.props` here?

Reply:
The args of `setNextState` defaults to `this.props`

Reply:
True, I hadn't paid attention. Or rather, the code wasn't clear enough. Could you please pass `this.props` here and have no default value in `setNextState`?